config/eurotronic/eur_spiritz.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product xmlns='http://code.google.com/p/open-zwave/'>
+    <CommandClass id="67" base="0" typeInterpretation="A" />
```

Comment:
The `typeInterpretation` attribute causes the xml validation to fail (`make xmltest`)